packages/svelte/src/internal/client/dom/elements/bindings/input.js

```diff
@@ -59,6 +62,7 @@ export function bind_value(input, get_value, update) {
 export function bind_group(inputs, group_index, input, get_value, update) {
 	var is_checkbox = input.getAttribute('type') === 'checkbox';
 	var binding_group = inputs;
+	var hydration_mismatch = false;
```

Comment:
this happened to me in other code paths, I think we need it here, too
```suggestion
	// needs to be let or related code isn't treeshaken out if it's always false
	let hydration_mismatch = false;
```